coalib/parsing/ConfParser.py

```diff
@@ -142,7 +142,8 @@ def __parse_lines(self, lines, origin):
                                 remove_empty_iter_elements=
                                 self.__remove_empty_iter_elements),
                                 # Stop ignoring
-                        allow_appending=(keys == []))
+                        allow_appending=(
+                            keys == [] or keys == [('', 'ignore')]))
```

Comment:
This is gobbly-de-gook.

I understand why `ignore` is here.

Why does keys contain `''` in this `+=` scenario?

Add some code comments here,, and possible put `[('', 'ignore')]` in a constant so you can give it a meaningful name and extra comments above it to explain what it is / why it is special.